ckanext/data_qld/auth_functions.py

```diff
@@ -34,3 +34,24 @@ def member_create(context, data_dict):
                         (str(user), group.id)}
     else:
         return {'success': True}
+
+
+def has_user_permission_for_some_org(context, data_dict):
+    user = context.get('user', '')
+    permission = data_dict.get('permission', '')
+    if authz.has_user_permission_for_some_org(user, permission):
+        return {'success': True}
+    else:
+        return {'success': False,
+                'msg': _('User {0} has no {1} permission for any organisation'.format(user, permission))}
+
+
+def has_user_permission_for_org(context, data_dict):
```

Comment:
@ThrawnCA - It is a dictionary because it is an Auth function registered via `IAuthFunctions` > `get_auth_functions` in the `reporting_plugin.py` file.

CKAN passes the arguments from the `toolkit.check_access` call into the auth function as a dictionary, not via named arguments.